ci/axis/miniforge-cuda-driver.yml

```diff
@@ -14,9 +14,7 @@ DOCKER_FILE:
 CUDA_VER:
   - 11.5
   - 11.4
-  - 11.3
   - 11.2
-  - 11.1
```

Comment:
CUDA 11.1 is associated with driver 455 (see table 3 [here](https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html)). so any mentions of driver 455 in this file should also be removed.